PW_CPV/README.md

```diff
@@ -0,0 +1,4 @@
+# Contents
+
+- `qes_current_develop.xsd`: tracks `develop` branch of https://github.com/QEF/q-e
```

Comment:
they track  branches on  https://gitlab.com/QEF/q-e ;  I know that this is just a formal distinguo but better to point to the main repo.